libclingo/clingo.h

```diff
@@ -3215,7 +3256,19 @@ CLINGO_VISIBILITY_DEFAULT bool clingo_control_is_conflicting(clingo_control_t *c
 //! @param[out] statistics the statistics object
 //! @return whether the call was successful; might set one of the following error codes:
 //! - ::clingo_error_bad_alloc
-CLINGO_VISIBILITY_DEFAULT bool clingo_control_statistics(clingo_control_t *control, clingo_statistics_t **statistics);
+CLINGO_VISIBILITY_DEFAULT bool clingo_control_statistics(clingo_control_t *control, const clingo_statistics_t **statistics);
+//! Add a callback to update user-defined statistics.
+//!
+//! See the @ref Statistics module for more information.
+//! Several callbacks supported.
+//!
+//! @param[in] control the target
+//! @param[in] cb a callback to the set_user_statistic function
+//! @param[in] data user data passed to the callback function
+//! @see ::clingo_set_user_statistics
+//! @return whether the call was successful
+CLINGO_VISIBILITY_DEFAULT bool clingo_control_add_user_statistics(clingo_control_t *control, clingo_set_user_statistics cb, void* data);
```

Comment:
I do not like the name `clingo_control_add_user_statistics`. In other places I used register instead of add. What about `clingo_control_register_user_statistics`.